apps/mobile-wallet/src/components/footers/FooterMenu.tsx

```diff
@@ -30,13 +30,14 @@ interface FooterMenuProps extends BottomTabBarProps {
   style?: StyleProp<ViewStyle>
 }
 
+const isIos = Platform.OS === 'ios'
```

Comment:
nit: we have this in other places too. Let's extract it to a util somewhere or just use `Platform.OS === 'ios'` which is short enough imo

Reply:
Done